backend/api/Areas/Property/Mapping/Parcel/ParcelParcelMap.cs

```diff
@@ -0,0 +1,86 @@
+using Mapster;
+using Pims.Api.Mapping.Converters;
+using System.Collections.Generic;
+using Entity = Pims.Dal.Entities;
+using Model = Pims.Api.Areas.Property.Models.Parcel;
+using BModel = Pims.Api.Models;
+using Pims.Dal.Helpers.Extensions;
+using System.Text.Json;
+using Microsoft.Extensions.Options;
+using System.Linq;
+
+namespace Pims.Api.Areas.Property.Mapping.Parcel
+{
+    public class ParcelParcelMap : IRegister
+    {
+        #region Variables
+        private readonly JsonSerializerOptions _serializerOptions;
+        #endregion
+
+        #region Constructors
+        /// <summary>
+        /// Creates a new instance of a ParcelParcelMap, initializes with specified arguments.
+        /// </summary>
+        /// <param name="serializerOptions"></param>
+        public ParcelParcelMap(IOptions<JsonSerializerOptions> serializerOptions)
+        {
+            _serializerOptions = serializerOptions.Value;
+        }
+        #endregion
+
+        public void Register(TypeAdapterConfig config)
+        {
+            //all mappings for mapping between a subdivision parcel that has divided parcels. Users cannot edit parcels/subdivisions through entity relationships so just pass ids.
+            config.NewConfig<Entity.Parcel, Model.SubdivisionParcelModel>()
```

Comment:
Since we have no intention of allowing users to modify parcels using the Parcels or Subdivisions association, there is no point in passing/mapping all of those fields since they won't be used.

Reply:
(So just use the ones that are going to be used, so pid/pin and id).